radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/transport/ExceptionHandler.java

```diff
@@ -109,8 +109,12 @@ private void handleFailure(Future<?> future, ChannelHandlerContext ctx) {
   }
 
   private void logAndCloseChannel(ChannelHandlerContext ctx, Throwable cause) {
-    var affectedEntity = mainHandler.map(Object::toString).orElse("channel");
-    log.warn("Exception on {} {}, closing channel to prevent resource leak", affectedEntity, cause);
+    if (log.isTraceEnabled()) {
+      var affectedEntity = mainHandler.map(Object::toString).orElse("(to a peer via proxy)");
+      log.trace("Closing channel {} due to {}", affectedEntity, cause);
```

Comment:
This one too https://stackoverflow.com/questions/7839565/logging-levels-logback-rule-of-thumb-to-assign-log-levels?noredirect=1&lq=1

Reply:
It's trace because the rest of the P2P messages (which aren't suggestive of bugs or misconfigurations) are trace.

There's an argument some of these messages should be DEBUG (or, most strictly, we should have a review of our peer to peer logging and what we log. I'd suggest EG we should actually log clearly at DEBUG when a connection is opened or closed to a peer at a give address), but this isn't part of this ticket I feel.

Reply:
I'll raise a separate ticket to revise our default logging level, logging configuration and logging strategy more generally.